frontend/src/features/entity_selector/EntitySelector.tsx

```diff
@@ -318,6 +358,8 @@ const EntitySelectorInnerAuth = ({
                 value={inputValue || uid || ''}
                 onChange={handleChange}
                 otherUid={otherUid}
+                historyId={historyId}
+                historyInsertion={lastSlide === 'up' ? 'right' : 'left'}
```

Comment:
After some manual testing, I am not convinced this behaviour is that intuitive.
I think I would prefer to make the previous video always accessible with a "down" swipe. That's also easier to explain: after selecting a video, swipe "up" to go the next suggestion, "down" to go back to the previous one.

Reply:
Good idea :+1: